pom.xml

```diff
@@ -698,6 +698,9 @@
 						<release>${maven.compiler.target}</release>
 						<source>${maven.compiler.source}</source>
 						<target>${maven.compiler.target}</target>
+						<compilerArgs>
+							<arg>-parameters</arg>
```

Comment:
With this parameter, the bytecode produced by `javac` will include parameter names so you can see actual parameter names in reflection and in some exceptions instead of just `arg0`, `arg1`, etc.
I added that because some JUnit errors gave me just an abstract `arg3` that couldn't be matched.

Having actual names also sometimes helps Spring, for argument-matching for some parts of the framework, such as matching REST controller parameters to method arguments without specifying the name in each `@RequestParm` annotation etc.

Reply:
Ah clear, good idea to add this. Thanks for the explanation. Should we add a comment that `-parameters` has that effect? (or later in a different PR)